dev/devicelab/bin/tasks/ios_content_validation_test.dart

```diff
@@ -119,6 +119,12 @@ Future<void> main() async {
           <String>['embed_and_thin'],
           environment: <String, String>{
             'SOURCE_ROOT': flutterProject.iosPath,
+            'BUILT_PRODUCTS_DIR': path.join(
+              flutterProject.rootPath,
+              'build',
+              'ios',
+              'Release-iphoneos',
+            ),
```

Comment:
This has been added to https://github.com/flutter/flutter/pull/69612.

This test fakes out some Xcode build settings to directly test xcode_backend.  We can always rely on `BUILT_PRODUCTS_DIR` being present when run directly in Xcode.